pkg/eventservice/event_broker.go

```diff
@@ -549,6 +553,7 @@ func (c *eventBroker) runSendMessageWorker(ctx context.Context, workerIndex int)
 	defer ticker.Stop()
 
 	resolvedTsCacheMap := make(map[node.ID]*resolvedTsCache)
+	redoResolvedTsCacheMap := make(map[node.ID]*resolvedTsCache)
```

Comment:
we can use one cacheMap to store both redo and normal resolvedTs. In this way, we can hide the redo/non-redo related logic among the cacheMap instead of in this function. It can help the code more readable